docs/documentation/advanced/term/fuzzy_term.mdx

```diff
@@ -42,3 +41,5 @@ SELECT * FROM search_idx.search(
   exempted from the fuzzy edit distance calculation, while false includes the
   entire string in the calculation.
 </ParamField>
+
+<Note>Highlighting is not currently supported for `fuzzy_term`.</Note>
```

Comment:
Why move these to the bottom? It seems to me like it would reduce visibility and users might be frustrated to implement and realize it's not working if they don't read the whole page